extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -683,14 +683,14 @@ private static Table readTableInternal(
             @NotNull final URI source,
             @NotNull final ParquetInstructions instructions) {
         final String scheme = source.getScheme();
-        if (scheme != null && !scheme.isEmpty() && !scheme.equals(FILE_URI_SCHEME)) {
+        if (scheme != null && !scheme.equals(FILE_URI_SCHEME)) {
             if (!scheme.equals(S3_URI_SCHEME)) {
                 throw new IllegalArgumentException(
                         "We only support reading single parquet file URI hosted on S3, but got " + source);
             }
             return readSingleFileTable(source, instructions);
         }
-        return readTableInternal(new File(source), instructions);
+        return readTableInternal(new File(source.getPath()), instructions);
```

Comment:
We shouldn't need to have these checks at this level I don't think... shouldn't the URI + plugin be enough to cover what is supported?